### DISCUSSION TITLE
How is data made available to `ios_unit_test` and `ios_ui_test`

### DISCUSSION THREAD

**Discussion by louwers** [bazelbuild/rules_apple#1991] [General]
The docs state about the `data` property:

> Files to be made available to the test during its execution. 

How are they made available?

**Comment by keith** [comment]
They're exposed through bazel's standard runfiles mechanism. That might be a bit weird for iOS tests so you might prefer to just include them as resources in the `data` property of your `swift_library` that contains the test code.

**Reply by louwers** [reply]
Yes or if a host app is used I can add it to that as well right?

**Comment by tinder-maxwellelliott** [comment]
This might help you out https://github.com/bazelbuild/rules_apple/blob/master/doc/common_info.md#tests

**Reply by louwers** [reply]
Thanks. 🙏 

Still using Objective-C but I guess the C API for runfiles can be used in that case.
